lib/server/swarm.js

```diff
@@ -29,15 +29,7 @@ function Swarm (infoHash, server) {
       peer_id: peer.peerId
     }
     self._onAnnounceStopped(params, peer, peer.peerId)
-
-    // When a websocket peer is evicted, and it's not in any other swarms, close
-    // the websocket to conserve server resources.
-    if (peer.socket && peer.socket.infoHashes.length === 0) {
-      try {
-        peer.socket.close()
-        peer.socket = null
-      } catch (err) {}
-    }
+    peer.socket = null
```

Comment:
You make a good point. It's not necessary, but I think it doesn't hurt to be explicit where we can. We could also set any references to `peer` to `null` but we don't keep any of those, other than what's in the LRU cache.